### PR TITLE
[RPGF-79] Add Project / Application Status

### PR DESCRIPTION
**Pull request by Billy19191**
# Description

[//]: # "Please include a summary of the changes and the related issue. (You can write below without deleting this comment)"
Add project / application status to all project Retro Funding 5

### What
[//]: # "Briefly describe this feature / fix / refactor (You can write below without deleting this comment)"
Enforce elgigbility in Retro Funding 5 card (all projects are eligible since Agora update their endpoint)

### Why
[//]: # "Describe the reason why we need this feature / fix / refactor or refer to Jira Ticket (You can write below without deleting this comment)"
To make it clear which project or application is qualify or diaqualify status for user to explore

## Type of change

Please check the relevent option with `x`.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (if needed)
- [x] My changes generate no new warnings
- [x] I have tested on my `Local Machine`

### CODE REVIEW

**Comment by Billy19191** [comment]
Note that, don't merge this PR until #43 is merged.

**Comment by wit03** [comment]
@Billy19191 Will review after #43 is merged.

File: app/(white_navbar)/explore/RetroPGF5/RetroType5.d.ts
Comment (Billy19191) [reply app/(white_navbar)/explore/RetroPGF5/RetroType5.d.ts:33]: Base on Agora API, this testimonial is string link not array. So, it's look good now!

File: app/(white_navbar)/explore/RetroPGF5/_component/Tab/ProjectTab.tsx
Comment (Billy19191) [reply app/(white_navbar)/explore/RetroPGF5/_component/Tab/ProjectTab.tsx:377]: Since all projects has already filter from Agora side, so we can force all projects status as true

File: app/(white_navbar)/explore/RetroPGF5/RetroType5.d.ts
Comment (Billy19191) [reply app/(white_navbar)/explore/RetroPGF5/RetroType5.d.ts:67]: Leave it as it is fine, since there's no project that have VC from Agora API, so we don't know the type of it.

**Comment by wit03** [comment]
#43 is merged. some conflict need to be resolved

**Review by wit03 — CHANGES_REQUESTED** [review]

**Review by wit03 — APPROVED** [review]
LGTM

**Pull request merged**
